exchanges/graphcache/src/types.ts

```diff
@@ -581,6 +581,17 @@ export type CacheExchangeOpts = {
    * the full keys docs.
    */
   keys?: KeyingConfig;
+  /** Enables global IDs for keying GraphQL types.
+   *
+   * @rematks
```

Comment:
```suggestion
   * @remarks
```

Reply:
Cheers! Nice catch 😅 time to put the linter back in